ledger/ledger-api-bench-tool/src/main/scala/com/daml/ledger/api/benchtool/SubmittedDataAnalyzing.scala

```diff
@@ -0,0 +1,98 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.api.benchtool
+
+import com.daml.ledger.api.benchtool.config.WorkflowConfig
+import com.daml.ledger.api.benchtool.services.PackageService
+import com.daml.ledger.api.benchtool.submission.BenchtoolTestsPackageInfo.BenchtoolTestsPackageName
+import com.daml.ledger.api.benchtool.submission.{
+  AllocatedParties,
+  BenchtoolTestsPackageInfo,
+  PartyAllocating,
+}
+import com.daml.ledger.api.v1.package_service.GetPackageResponse
+import com.daml.lf.archive.{ArchivePayloadParser, Decode, Reader}
+import com.daml.lf.data.Ref
+import com.daml.lf.language.Ast
+import org.slf4j.{Logger, LoggerFactory}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+/** Contains utilities for retrieving useful facts
+  * from data already submitted to a Ledger API server.
+  * (The motivating use case are the benchmarks that do not perform a submission step on their own
+  * and for that reason cannot statically determine these facts.)
+  */
+object SubmittedDataAnalyzing {
```

Comment:
How about just `SubmittedData` ?

Reply:
But this class is no the submitted data; so maybe `SubmittedDataUtils`?